test/test_torch.py

```diff
@@ -11169,80 +11169,97 @@ def test_signbit_non_boolean_output(self, device, dtype):
             torch.signbit(t, out=out)
 
     def test_logical_any(self, device):
```

Comment:
For these tests, would it be possible test both of them in the same test vs. their NumPy counterparts? 

That is, using the @dtypes decorator, enumerating the functions, passing multiple random inputs through them, and then comparing them with NumPy. The values in the random inputs should include zeros, -inf, inf, and NaNs, too, to validate those are handled the same way NumPy handles them. The random input shapes should include 0D (empty), 1D, and 3D tensors. 

It's important we thoroughly enumerate the test cases programmatically to ensure we're compatible with NumPy.

Reply:
Instead of touching these tests themselves, I have added a test with `keepdim` in `test_all_any_vs_numpy` which tests with 0D to 4D tensors with random inputs including 0, -inf, inf, etc and compares againts NumPy.

I think these tests also make sense (to verify against known output) so I am slightly apprehensive in removing them.
Do let me know how you feel.